### ISSUE TITLE
iOS - Play crashes when loading ISO

### ISSUE THREAD

**Issue by firelad97** [jpd002/Play-#1017]
As of iOS 14.4, JIT was removed and it makes Play crash when loading ISO. Is there not a way we can change recompiler/CPU for iOS (ie, IR Interpreter)?

I can't access files on non-Jailbroken iPhone for some reason to check configuration file.

**Comment by jpd002** [comment]
Yeah, the situation really sucks, I was hopeful when it worked again in 14.2 after being disabled, but I guess Apple really doesn't want JIT on iOS afterall.

There's no option to use an interpreter atm, but maybe it's something we should consider. I really don't know how it will perform though, I'm not sure it's going to be a viable option.

**Comment by firelad97** [comment]
I think you should consider it. Gives us more options to play with while maintaining stable emulation. :)

If I'm right, using an interpreter probably will only work fine with most 2D games, if there are any that already are working. It can't hurt to try though, as most other iOS emulators have that option. A good iPhone CPU chip might will be needed for this - for example I have A12. It is still pretty powerful for a phone.

**Comment by MLGESUS5183** [comment]
I have the issue on 14.3 for the libretro core

**Comment by hujiba** [comment]
Also having this issue on the libretro core on iOS 15 beta. Any plans to fix it?
